lib/traject/extract_barcode.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+require 'traject/extraction_tools'
+extend ExtractionTools
+
+module ExtractBarcode
+  def extract_barcode
```

Comment:
Is a customized extraction method needed here? As far as I can tell, you can replace this method with `extract_marc("997a"), trim_punctuation` and get the same results. Did you try that before making this method?